src/custom/theme/components.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { HTMLProps } from 'react'
+import { StyledLink, handleClickExternalLink } from './baseTheme'
+import { externalLinkAnalytics } from 'utils/analytics'
+
+export * from '@src/theme/components'
```

Comment:
🚨 Path police! 🚨

```suggestion
export * from 'theme/components'
```

Unless you actually need it from base, then just ignore me :)

Reply:
It was a good comment, but I think in this case he wants to re-export all the original code, because he is actually doing a "Mod"